### PR TITLE
fix rolematrix chips/filter padding/button styles, XM1-35146

### PR DESCRIPTION
**Pull request by mak7iliano**

### CODE REVIEW

File: packages/administration/roles-matrix/roles-matrix.component.html [hunk 1]
```diff
@@ -106,14 +106,10 @@ <h4 mat-card-title>{{'rolesManagement.matrix.title'|translate}}</h4>
             <ng-container *ngFor="let role of matrix?.roles; index as i">
               <ng-container [matColumnDef]="role">
                 <th *matHeaderCellDef scope="col" mat-header-cell>
-                  <mat-chip-grid #gridRoles class="mt-2 mb-2 d-block">
```
Comment (zverbeta) [inline packages/administration/roles-matrix/roles-matrix.component.html:109]: Is mat-chip-grid deprecated?

Comment (mak7iliano) [reply packages/administration/roles-matrix/roles-matrix.component.html:109]: this is need to use chip without text input

Comment (zverbeta) [reply packages/administration/roles-matrix/roles-matrix.component.html:109]: If we can remove chips, how can we add those chips back?

File: packages/administration/roles-matrix/roles-matrix.component.html [hunk 2]
```diff
@@ -106,14 +106,10 @@ <h4 mat-card-title>{{'rolesManagement.matrix.title'|translate}}</h4>
...
-                    <mat-chip-row (removed)="onHideRole(role, i)" removable>
-                      {{role}}
-                      <mat-icon matChipRemove>cancel</mat-icon>
-                    </mat-chip-row>
-
-                    <input [matChipInputFor]="gridRoles" />
```
[the remark this answers is not in the document]
Comment (mak7iliano) [reply packages/administration/roles-matrix/roles-matrix.component.html:115]: I thought that input was not in use, cause there is no code to process him

[the remark this answers is not in the document]
Comment (zverbeta) [reply packages/administration/roles-matrix/roles-matrix.component.html:115]: We use it as a ref to input https://github.com/xm-online/xm-webapp/pull/2024/files/4957bf3bcd4b4088c611112de97de2227fd32aba#diff-260417e4c246a8391b7ea078ac550ee9dc8f186b6c35bf354cf20589cd34a8c6L115

[the remark this answers is not in the document]
Comment (zverbeta) [reply packages/administration/roles-matrix/roles-matrix.component.html:115]: I see

File: packages/administration/roles-matrix/roles-matrix.component.html (hunk 1 above)
Comment (mak7iliano) [reply packages/administration/roles-matrix/roles-matrix.component.html:109]: deleted chip appears above

Comment (zverbeta) [reply packages/administration/roles-matrix/roles-matrix.component.html:109]: Could we also remove matChipRemove and icon which remove the chip?

Comment (zverbeta) [reply packages/administration/roles-matrix/roles-matrix.component.html:109]: It looks like we can remove chips, but can't add again

**Review by zverbeta — APPROVED** [review]

**Review by Kaskyi — APPROVED** [review]

**Pull request merged**